src/scripting/flash/system/flashsystem.cpp

```diff
@@ -1234,7 +1238,11 @@ ASFUNCTIONBODY_ATOM(System,totalMemory)
 	char* buf=nullptr;
 	size_t size=0;
 	FILE* f = open_memstream(&buf, &size);
+	#ifndef __APPLE__
 	if (!f || malloc_info(0,f)!=0)
+	#else
+	if (!f)
+	#endif
```

Comment:
This would lead to always returning 0 for for System.totalMemory on MacOS.
I think it would be better to do it similar to Windows with a LOG_NOT_IMPLEMENTED message and returning 1024 (see lines 1229ff in this file)